internal/installercache/installercache.go

```diff
@@ -8,19 +8,22 @@ import (
 	"path/filepath"
 	"strings"
 	"sync"
+	"syscall"
 	"time"
 
+	"github.com/alecthomas/units"
```

Comment:
Would it be better to use a more widely accepted library for this?
Maybe `k8s.io/apimachinery/pkg/api/resource`?

Reply:
Ah okay, I didn't know this one was already in use. That's fine then.